index.md

```diff
@@ -15,6 +15,7 @@ Welcome to the Meilisearch documentation. Here you'll find everything you need t
 - [Deploy Meilisearch on AWS](/learn/cookbooks/aws.md)
 - [Deploy Meilisearch on GCP](/learn/cookbooks/gcp.md)
 - [Deploy Meilisearch on DigitalOcean](/learn/cookbooks/digitalocean_droplet.md)
+- [Deploy Meilisearch on Koyeb](/learn/cookbooks/koyeb.md)
```

Comment:
Hi @edouardb ! Do you think you could also remove this line when you update this PR?

The deploy links on our homepage are only intended to cover the top 3 or 4 industry-standard cloud providers. Adding Koyeb to that list might mean adding others in the future, and that could make our homepage very crowded. I hope you understand—your guide will still be accessible from the sidebar in the Learn section of the site.

Thank you very much!

```suggestion
```